src/app/market/sell/add-item/add-item.component.ts

```diff
@@ -354,12 +356,16 @@ export class AddItemComponent implements OnInit, OnDestroy {
     this.upsert().then(t => {
       this.modals.unlock({timeout: 30}, (status) => {
         this.template.post(t, 1).toPromise().then(listing => {
-        this.snackbar.open('Succesfully added Listing!')
-        console.log(listing);
-        this.backToSell();
+          this.isInProcess = false;
+          this.snackbar.open('Succesfully added Listing!')
+          console.log(listing);
```

Comment:
Can this be wrapped instead with a debug log line if it is necessary to output the listing information?